### PR TITLE
Add Disguise Support

### LINKED ISSUE #1793: Add /uall and /dtoggle
Mark recently added Libsdisguises to the server and we need /uall and /dtoggle I am re-adding those commands now

### PR DESCRIPTION
**Pull request by lemonsked**
Fixes #1793,
@CreeperSeth, I fixed /dtoggle.
Dependency link:https://www.dropbox.com/s/kvnvjqj0nxvm3oo/LibsDisguises.jar?dl=0

### CODE REVIEW

**Comment by CreeperSeth** [comment]
@JeromSar this should work, I've tested this and it works like a charm.

**Comment by robingall2910** [comment]
#1800 congrats boys :clap:

**Comment by OxLemonxO** [comment]
This update is already added as a patch on the TF server

File: src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        }
+        return null;
+    }
+    
+    public void enableDisguises(Boolean state)
+    {
+      final LibsDisguises libsDisguises = getLibsDisguisesPlugin();
+      
+      if (libsDisguises == null)
+       {
```
Comment (JeromSar) [inline src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java]: Formatting.

File: src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            }
+        }
+        catch (Exception ex)
+        {
+            FLog.severe(ex);
+        }
+        return null;
+    }
+    
+    public void enableDisguises(Boolean state)
```
Comment (JeromSar) [inline src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java]: Why not `boolean`?

File: src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        {
+            DisguiseAPI.enableDisguises();
+        }
+        else
+        {
+            DisguiseAPI.disableDisguises();
+        }
+    }
+    
+    public void undisguiseAll(Boolean admins)
```
Comment (JeromSar) [inline src/main/java/me/totalfreedom/totalfreedommod/bridge/LibsDisguisesBridge.java]: `boolean`?

**Comment by JeromSar** [comment]
Looks very good on initial review. Still need to test this. :)

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_undisguiseall.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+@CommandPermissions(level = Rank.SUPER_ADMIN, source = SourceType.BOTH)
+@CommandParameters(description = "Undisguise all players on the server", usage = "/<command>", aliases = "uall")
+public class Command_undisguiseall extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (DisallowedDisguises.disabled)
```
Comment (JeromSar) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_undisguiseall.java]: Won't this throw an error when LibsDisguises is not enabled on the server? You could instead check if `ldb.getLibsDisguisePlugin()` is null or not.

**Comment by OxLemonxO** [comment]
@JeromSar Fixed those issues.

**Comment by JeromSar** [comment]
Merged in https://github.com/TotalFreedom/TotalFreedomMod/commit/cadf4e4fc7d51da72f1dd3424d61ef31d1a4ff08

**Comment by JeromSar** [comment]
There were some issues. Looking into this now.

**Comment by JeromSar** [comment]
Reverted changes to /dtoggle and merged in https://github.com/TotalFreedom/TotalFreedomMod/commit/2008871658e1bd127dbdd8adbbdb8bd7cc3274d0

**Pull request closed without merging**